test/spec.index.js

```diff
@@ -250,6 +250,57 @@ describe('Template Mixins', function () {
                 }));
             });
 
+            it('by default, assumes the field isn\'t required', function () {
```

Comment:
Well quite. It's ugly to change the form-wizard functionality and create an 'anti-validator' though it could be done.

My personal opinion is that this PR should be merged and we should default to fields being optional from a technological standpoint, and put the onus back on development teams to only ask for the information they need. (Which IMHO is where it belongs. The libraries should reflect the nature of the web and not the nature of current government policies. That should be enforced at another level.)

BTW, presumably all projects using these libraries suffer from the bug reported on BRP and this PR should fix it for them. Which is pretty nice.

There's also the issue that enforcing government policy in the libraries themselves makes them far less useful outside of government. - I don't see any reason why a lot of this stuff shouldn't be used out in the wider world!